packages/api/endpoints/rules.js

```diff
@@ -129,7 +130,7 @@ async function put({ params: { name }, body }, res) {
 
     await dbClient.transaction(async (trx) => {
       await trx(tableNames.rules)
-        .insert(newPostgresRecord)
+        .insert(omit(newPostgresRecord, 'created_at'))
```

Comment:
So is the outcome from yesterday's conversation that we also want to omit `updated_at` here?

Reply:
Not entirely - there's a bit of a rabbit hole here in the possible differences between RDS/dynamo writes.  I'm going to dig into that and re-issue this PR once I have consistency sorted.